Documentation/Books/AQL/Functions/TypeCast.md

```diff
@@ -14,6 +14,9 @@ Each of the these functions takes an operand of any data type and returns a resu
 value with the type corresponding to the function name. For example, *TO_NUMBER()*
 will return a numeric value.
 
+Type casting functions
+----------------------
+
 ### TO_BOOL()
```

Comment:
https://docs.arangodb.com/devel/AQL/Functions/TypeCast.html#type-check-functions should become its own toplevel file later on, but not now. Then headlines of the functions would be similar, since you don't have another section.